modules/mod_feed/tmpl/default.php

```diff
@@ -56,7 +56,7 @@
 			<?php if (!empty($feed)) : ?>
 				<ul class="newsfeed<?php echo $params->get('moduleclass_sfx'); ?>">
 					<?php for ($i = 0, $max = min(count($feed), $params->get('rssitems', 5)); $i < $max; $i++) : ?>
-						<?php $uri   = (!empty($feed[$i]->uri) || $feed[$i]->uri !== null) ? trim($feed[$i]->uri) : trim($feed[$i]->guid); ?>
+						<?php $uri   = !empty($feed[$i]->uri) || $feed[$i]->uri !== null ? trim($feed[$i]->uri) : trim($feed[$i]->guid); ?>
```

Comment:
hmm can you revert that change please as that is not simple codestyle change? Thnaks